packages/jest-config/src/ValidConfig.js

```diff
@@ -40,7 +40,10 @@ export default ({
       statements: 100,
     },
   },
+  cwd: '/root',
   dependencyExtractor: '<rootDir>/dependencyExtractor.js',
+  detectLeaks: false,
```

Comment:
these are only cli flags, not sure if they make sense here (or in `Defaults`)

Reply:
We don't have `runInBand` here, so `detectLeaks` and `detectOpenHandles` shouldn't be added as well

Reply:
cool, removed them